### PR TITLE
Adding Reg powers sentence to confirmation screen.

### PR DESCRIPTION
**Pull request by clewis1**
BI-5555

### CODE REVIEW

File: src/main/resources/templates/confirmation.html [hunk 1]
```diff
@@ -25,8 +25,14 @@ <h1 class="govuk-panel__title" th:text="#{confirmation.title}"></h1>
...
                 <p class="govuk-body" th:text="#{confirmation.next.body1}"></p>
-                <p class="govuk-body" th:text="#{confirmation.next.body2}"></p>
-                <p class="govuk-body" th:text="#{confirmation.next.body3}"></p>
+                <div class="govuk-body" th:if="${registrarsPowers}">
+                    <p th:text="#{confirmation.next.body2}"></p>
+                </div>
+                <div class="govuk-body" th:unless="${registrarsPowers}">
+                    <p th:text="#{confirmation.next.body3}"></p>
+                </div>
+                <div th:unless="${showCcReminder}" th:if="*{maximumUploadsAllowed} == 1" class="govuk-inset-text"
```
Comment (PhilipGlover) [inline src/main/resources/templates/confirmation.html:34]: is this showCcReminder text relevant here?  this is from the Document Upload screen isn't it?

Comment (clewis1) [reply src/main/resources/templates/confirmation.html:34]: Not sure how this got in! Must be a copied example. Will remove.

**Comment by PhilipGlover** [comment]
I've run the code and confirmation screen displays as expected.  I note the sonar review has picked up some possible code changes

**Review by PhilipGlover — COMMENTED** [review]
Locally confirmation screen displays as expected. 
Sonar review code smells also dealt with.

**Review by PhilipGlover — APPROVED** [review]

**Pull request merged**